Syntaxes/Makefile.plist

```diff
@@ -489,7 +489,7 @@
 				</dict>
 			</dict>
 			<key>end</key>
-			<string>^(?!\t)</string>
+			<string>[^\\]$</string>
```

Comment:
With this change added in I get the `meta.scope.target` ending before the `meta.scope.recipe` begins. The logic for extending the pre-requisite section should be contained in `meta.scope.prerequisites` rather than it's parent rule as that is what is really the cause here.

Reply:
I'll check it.

Reply:
Really, with my change target ending before the recipe begins. But in previous version the worse thing was happen: target ended with the first TAB inside the recipe.

Actually, I didn't change a lot previous logic, but I handled the following issue: TAB INSIDE MULTILINE PREREQUISITES also meant the end of prerequisites and begin of recipe: it's wrong in a case of multiline preresuisites ("\\"+"\n"): in a case of multiline prerequisites TAB isn't treat as a start of recipe.

Reply:
I have rewritten how prerequisites are matched entirely, should be corrected now.